recipes/qdbm/all/CMakeLists.txt

```diff
@@ -0,0 +1,83 @@
+cmake_minimum_required(VERSION 3.1)
```

Comment:
Could you please explain your motivation for writing your own CMake file instead of using the upstream build system?

Reply:
Sure, it's much easier to hook into the build from Conan if it's a CMake build instead of an Autotools gobbledygook. It also builds trivially across every platform.

Also, I rather spend a couple hours writing a CML that I can guarantee to work correctly than spend multiple days trying to untangle some Autotools mess that I hope works correctly. See the net-snmp for an example where I'm not sure things work properly despite the amounts of bandaids I had to apply.

Reply:
We avoid creating extra files for multiple reasons. First, maintenance, one file more requires more effort. Second, expected behavior, the autotools can be a mess, but it results a file with an expected behavior, if we provide cmake or any other file and forget any flag, it will result in a different artifact (libs, bin, ...).
There are some cases where we use autootools for gcc and clang, but Visual Studio we really need a custom cmake file, but it's an exception.
Said that, please, keep autotools as the first option, if you get stuck, you still can ask for help here.

Reply:
I'd contest that by pointing out that QDBM hasn't seen an update since 2005 and CMake works better on all platforms, so there is nothing to maintain in that regard and the conanfile is much simpler and shorter than it would otherwise be, which makes it easier to maintain. On a second look, I will give you that maybe the `.def` file (which I took it from the [PHP repository](https://github.com/winlibs/qdbm/blob/master/qdbm.def)) file should be dropped, because there are already export/import attributes in the source code.